### PR TITLE
ISAICP-3619: header bar improvements

### PR DESCRIPTION
**Pull request by luke-waat**
Rebuild is required.

### CODE REVIEW

File: web/modules/custom/joinup_core/templates/statistics-block.html.twig [hunk 1]
```diff
@@ -33,6 +23,16 @@
     </a>
     <div class="stats__description">Collaborate around thematics</div>
   </div>
+  <div class="stats__wrapper stats__wrapper__solutions">
```
Comment (kalinchernev) [inline web/modules/custom/joinup_core/templates/statistics-block.html.twig]: Please include a translate filter to strings in this template to make them accessible for localization.

File: web/themes/joinup/prototype/sass/components/_search-bar.sass [hunk 2]
```diff
@@ -57,39 +55,57 @@
     color: lighten($color-grey-medium, 10%)
 
 .search-bar__input-container
-  margin-left: 40px
+  //width: calc(100% - 50px)
```
Comment (kalinchernev) [inline web/themes/joinup/prototype/sass/components/_search-bar.sass]: Unnecessary lines could be removed.

File: web/modules/custom/joinup_core/templates/statistics-block.html.twig (hunk 1 above)
Comment (luke-waat) [inline web/modules/custom/joinup_core/templates/statistics-block.html.twig]: Done.

File: web/themes/joinup/prototype/sass/components/_search-bar.sass (hunk 2 above)
Comment (luke-waat) [inline web/themes/joinup/prototype/sass/components/_search-bar.sass]: Done.

**Pull request merged**